core/src/it/scala/org/apache/pekko/persistence/jdbc/integration/ScalaJdbcDurableStateChangesByTagTest.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
```

Comment:
* new files should be given the standard ASF header
* This `derived from Akka` header is just for files where an important part of the code is copied from an existing Akka file or a Pekko file that has the `derived from Akka` header.